package.json

```diff
@@ -12,6 +12,7 @@
     "local:react": "REACT_APP_CONTEXT=true NODE_ENV=dev react-scripts start",
     "local:airtable-sync": "NODE_ENV=dev node -r module-alias/register ./src/workers/airtable-sync/worker.js",
     "local:script": "DOTENV_CONFIG_PATH=./.env node -r module-alias/register -r dotenv/config -r ~strings/i18nextInit",
+    "local:twilio": "twilio-run start --ngrok='' --port 3001 src/twilio",
```

Comment:
@mab253 - Made a small tweak to the ngrok port so we can run webapp and twilio locally at the same time, otherwise worked perfectly! :ok_hand: